commons-math-legacy/src/main/java/org/apache/commons/math4/legacy/fitting/SimpleCurveFitter.java

```diff
@@ -187,10 +187,7 @@ public int compare(WeightedObservedPoint p1,
                         return comp;
                     }
                     comp = Double.compare(p1.getWeight(), p2.getWeight());
```

Comment:
No need to store the result in `comp`. Just return the result direct.

Reply:
yes, inline variable.
TY